### DISCUSSION TITLE
HTML DuckDuckGo

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2280] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

- Search bar & button
- Text & dialog colors
- Link colors & background

### Any additional comments?

The site doesn't use any scripts or (to my knowledge) styles, so there isn't much to change.

### Submission Guidelines

- [ ] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Have you tried [our DuckDuckGo userstyle](https://github.com/catppuccin/userstyles/tree/main/styles/duckduckgo)?

**Reply by ghost** [reply]
Yes, but it isn't working for some reason. Maybe if I can figure out why, I'll be able to just add `html.duckduckgo.com` to the list of domains
